src/components/Products.jsx

```diff
@@ -2,21 +2,20 @@ import React, { useState, useEffect } from "react";
 import { Card, Item, Container, List } from "semantic-ui-react";
 import { ToastContainer, toast } from "react-toastify";
 import axios from "axios";
-import "react-toastify/dist/ReactToastify.css";
 
 const Products = () => {
   const [products, setProducts] = useState([]);
 
   const fetchProducts = async () => {
-    const response = await axios.get("https:/api/products");
+    const response = await axios.get("https:reqres.in/api/products");
```

Comment:
Why is the reqres.in added to the https: address

Reply:
if no reqres, the Axios will not work you can pull from me and try run cypress. after removing reqres the userCanSeeAnApplication.feature will not pass